src/Vendr.PaymentProviders.Opayo/Helpers/OpayoInputLoader.cs

```diff
@@ -57,23 +57,32 @@ private static void LoadOrderValues(Dictionary<string, string> inputFields, Orde
             LoadShippingDetails(inputFields, order, settings, context);
 
             if (settings.DisplayOrderLines)
-                LoadOrderLines(inputFields, order);
+                LoadOrderLines(inputFields, order, settings);
 
         }
 
-        private static void LoadOrderLines(Dictionary<string, string> inputFields, OrderReadOnly order)
+        private static void LoadOrderLines(Dictionary<string, string> inputFields, OrderReadOnly order, OpayoSettings settings)
         {
             var orderLines = new List<string>();
             foreach(var item in order.OrderLines)
             {
-                orderLines.Add($"{item.ProductReference}:{item.Quantity}:{item.UnitPrice.Value.WithoutTax:0.00}:{item.UnitPrice.Value.Tax:0.00}:{item.UnitPrice.Value.WithTax:0.00}:{item.TotalPrice.Value.WithTax:0.00}");
+                var itemDescription = GetItemDescriptionByOrderPropertyDescriptionAlias(item, settings.OrderLineDescriptionPropertyAlias);
+                orderLines.Add($"{itemDescription}:{item.Quantity}:{item.UnitPrice.Value.WithoutTax:0.00}:{item.UnitPrice.Value.Tax:0.00}:{item.UnitPrice.Value.WithTax:0.00}:{item.TotalPrice.Value.WithTax:0.00}");
             }
 
             orderLines.Insert(0, orderLines.Count.ToString());
 
             inputFields.Add(OpayoConstants.TransactionRequestFields.Basket, string.Join(":", orderLines));
         }
 
+        private static string GetItemDescriptionByOrderPropertyDescriptionAlias(OrderLineReadOnly lineItem, string alias)
+        {
+            if (string.IsNullOrEmpty(alias)) return $"{lineItem.Name} ({lineItem.Sku})";
```

Comment:
Maybe store the default description in a variable so we don't have to do string interpolation twice. 

No idea if that really makes much of a difference mind.

Reply:
As you say, probably doesn't make much difference but good practice nonetheless, so have altered this so it is initially set in a variable.